src/app/routes/api/v3/admin/forms/admin-forms.form.routes.ts

```diff
@@ -32,6 +32,18 @@ AdminFormsFormRouter.route('/')
    */
   .post(AdminFormController.handleCreateForm)
 
+AdminFormsFormRouter.route('/owned')
```

Comment:
semantically `/mine` would probably be a better name for the route!

Reply:
Done!